api/src/pcapi/scripts/pro/upload_reimbursement_csv_to_offerer_drive.py

```diff
@@ -78,14 +78,15 @@ def _create_and_get_csv_file(user_id: int, batch_id: int, parent_folder_id: str,
         )
         return None
 
-    reimbursement_details = find_reimbursement_details_by_invoices([invoice.reference for invoice in invoices])
-    csv_lines = [reimbursement_detail.as_csv_row() for reimbursement_detail in reimbursement_details]
     headers = ReimbursementDetails.get_csv_headers()
     local_path = pathlib.Path(tempfile.mkdtemp()) / filename
-    with open(local_path, "w+", encoding="utf-8") as fp:
+    with open(local_path, "a+", encoding="utf-8") as fp:
         writer = csv.writer(fp, dialect=csv.excel, delimiter=";", quoting=csv.QUOTE_NONNUMERIC)
         writer.writerow(headers)
-        writer.writerows(csv_lines)
+        for invoice in invoices:
+            reimbursement_details = find_reimbursement_details_by_invoices([invoice.reference])
+            csv_lines = [reimbursement_detail.as_csv_row() for reimbursement_detail in reimbursement_details]
+            writer.writerows(csv_lines)
```

Comment:
tu pourrais encore optimiser un peu pour la ram avec 
```suggestion
            csv_lines = [reimbursement_detail.as_csv_row() for reimbursement_detail in reimbursement_details]
            for reimbursement_detail in reimbursement_details:
                writer.writerow(reimbursement_detail.as_csv_row())
```